### ISSUE TITLE
Lack of zaqarclient in the requiremnets

### ISSUE THREAD

**Issue by apolloliu** [StackStorm-Exchange/stackstorm-openstack#16]
Actually this tests couldn't go through without zaqarclient. When I commit my PR, the error occurred:
```
==================== pylint ====================

. virtualenv/bin/activate; if [ ! "actions/src/lib/base.py actions/src/lib/utils.py" ]; then echo "No files have changed, skipping run..."; else (REQUIREMENTS_DIR=/home/ubuntu/ci/.circle/ CONFIG_DIR=/home/ubuntu/ci/lint-configs/ st2-check-pylint-pack /home/ubuntu/stackstorm-openstack || exit 1); fi;
Running pylint on pack: stackstorm-openstack
--> No pylint issues found in actions.
************* Module messaging_sensor
E:  2, 0: Unable to import 'zaqarclient.queues.v1' (import-error)
make: *** [.pylint] Error 1
```

**Comment by LindsayHill** [comment]
What happens if you add that to requirements.txt in your PR?

**Comment by apolloliu** [comment]
Thanks @LindsayHill , I didn't add that into my PR and this requirement isn't related to my PR. But it's reported as above. In addition, I found zaqarclient has been imported in the [code](https://github.com/StackStorm-Exchange/stackstorm-openstack/blob/master/sensors/messaging_sensor.py#L2). How do think about this? If I was correct, I would commit another PR #18 to fix it.

**Comment by LindsayHill** [comment]
Resolved by #19
